roofit/roofitcore/inc/RooAbsPdf.h

```diff
@@ -71,6 +71,7 @@ class RooAbsPdf : public RooAbsReal {
   RooDataSet *generate(const RooArgSet &whatVars, const RooDataSet &prototype, Int_t nEvents= 0,
              bool verbose=false, bool randProtoOrder=false, bool resampleProto=false) const;
 
+  std::string translate(RooFit::Detail::ADContextManager & /*ctxManager*/) const override;
```

Comment:
```suggestion
  std::string translate(RooFit::Detail::ADContextManager & ctxManager) const override;
```
No need to comment out the parameter name if this is only the function declaration.